front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -11,6 +12,7 @@ import {
 
 import useFeatureFlag from 'hooks/useFeatureFlag';
 
+import Divider from 'components/admin/Divider';
```

Comment:
Maybe we should move the divider component to the component library, if it's not used only in the admin 🤔

Reply:
Makes sense! I'll make a separate PR for that.